aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,90 +1,65 @@
-  href="/components/datatable#paginated"
-  src="/pagination-table-preview.png"
-  alt="HPE Design System Paginated Table Example"
-/>
+| Label | Region          | Purpose                                                                            | Required | Notes                                                                                                                                      |
+| :---: | --------------- | ---------------------------------------------------------------------------------- | :------: | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Container**   | Container, organizes its children, where they appear, how they adapt responsively. | Required | --                                                                                                                                         |
+| **2** | **Summary**     | Provide clarity on what range of items are in view.                                | Optional | If no label is shown, icon is required.                                                                                                    |
+| **3** | **StepOptions** | Displays the current number of items within a page and allows the user to modify.  | Optional | Step defaults to 10. Dropdown options default to 10, 25, 50, 100. The default step and step options can be customized to fit the use case. |
+| **4** | **Navigation**  | Allow user to jump to a specific page.                                             | Required | a11yTitle required. If no icon is shown, label is required.                                                                                |
 
-### Paginated list
+## Types
 
-Lists often contain large amounts of data. To improve the efficiency that a user can navigate to a specific item or range of items, use pagination.
+### Pagination with Standard Navigation
 
-<ExampleImagePreview
-  label="See Paginated List"
-  href="/templates/lists#paginated"
-  src="/pagination-list-preview.png"
-  alt="HPE Design System Paginated List Example"
-/>
+The standard navigation in the Pagination component displays up to 7 buttons, including Edge Pages (first and last), Middle Pages (maximum of 3 pages) and Arrow Buttons (previous and next).
 
-### Paginated cards
+### Pagination with Custom Navigation
 
-<Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/pagination/PaginationCardsExample.js"
-  docs="https://v2.grommet.io/pagination?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <PaginationCardsExample />
-</Example>
+The navigation in the Pagination component can be customized to better fit different use cases. The following settings can be adjusted:
+Middle pages: adjusts the number of visible pages between the edge pages. Edge pages: adjusts the number of visible edge pages.
```

Comment:
[Middle pages](https://v2.grommet.io/pagination?theme=hpe#numberMiddlePagesl): adjusts the number of visible pages between the edge pages. 
[Edge pages](https://v2.grommet.io/pagination?theme=hpe#numberEdgePages): adjusts the number of visible edge pages.